fishy/gui/config_top.py

```diff
@@ -88,21 +88,30 @@ def toggle_collect():
     borderless = Checkbutton(controls_frame, var=BooleanVar(value=config.get("borderless")))
     borderless.grid(row=1, column=1)
 
+    def del_action_key(event):
+        action_key_entry.delete(0,"end")
+        action_key_entry.insert(0, str(event.char))
+
     Label(controls_frame, text="Action Key:").grid(row=2, column=0)
     action_key_entry = Entry(controls_frame, justify=CENTER)
     action_key_entry.grid(row=2, column=1)
     action_key_entry.insert(0, config.get("action_key", "e"))
-
+    action_key_entry.bind("<KeyRelease>", del_action_key)
 
     Label(controls_frame, text="Auto-Looting: ").grid(row=3, column=0, pady=(15, 0))
     collect_allow_auto = Checkbutton(controls_frame, command=toggle_collect, var=BooleanVar(value=config.get("collect_allow_auto")))
     collect_allow_auto.grid(row=3, column=1, pady=(15, 0))
-    
+
+    def del_collect_key(event):
```

Comment:
del_action_key does the same thing, try to use single function for both the callbacks, to reduce code repetition

Reply:
I'd have to call the function with an argument giving the correct entry. Don't know how. Can I just add arguments to the callback?

Reply:
use lambda or partial functions